assets/js/blocks/attribute-filter/edit.js

```diff
@@ -237,7 +237,7 @@ const Edit = ( { attributes, setAttributes, debouncedSpeak } ) => {
 				) }
 			</p>
 			<Button
-				className="wc-block-attribute-filter__add_attribute_button"
+				className="wc-block-attribute-filter__add-attribute-button"
```

Comment:
Not related to this pull, but I took the chance to change this class name so it follows BEM convention.

Reply:
This is released code though, so we might want to make sure it's noted in the dev notes/changelog for the next release.

Reply:
Those classes (`wc-block-attribute-filter__add-attribute-button` and `wc-block-price-slider__add-product-button`) are only used in the editor, they refer to the button in this placeholder:
![Screenshot from 2020-05-21 15-58-13](https://user-images.githubusercontent.com/3616980/82652185-78a0a500-9c1d-11ea-8b0e-2c75724fdef0.png)

In that case, when an element is only displayed in the editor, do we need dev notes too?

Reply:
We shouldn't do - this is internal.